app/ai_backend/genesis_connector.py

```
@@ -83,3 +115,451 @@
+            self._send_error_response(f"Response serialization failed: {e}")
+    
+    def _send_error_response(self, error_message):
+        """Send error response"""
+        error_response = {
+            "success": False,
+            "persona": "error",
+            "result": {"error": error_message}
+        }
+        try:
+            print(json.dumps(error_response), flush=True)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)